Sources/Plasma/NucleusLib/pnNetCli/pnNcCli.cpp

```diff
@@ -356,11 +366,18 @@ static void BufferedSendData (
                 // we reserve one space for the NULL terminator
                 const uint16_t length = (uint16_t) std::char_traits<char16_t>::length((const char16_t *) *msg);
                 ASSERT_MSG_VALID(length < cmd->count);
+
                 // Write actual string length
                 uint16_t size = hsToLE16(length);
                 AddToSendBuffer(cli, sizeof(uint16_t), (const void*)&size);
+
+                std::vector<char16_t> temp(length);
```

Comment:
Why not `std::unique_ptr<char16_t[]>` for consistency with the rest of the pnNcCli buffers?

Reply:
This is what Zrax suggested here: https://github.com/H-uru/Plasma/pull/1473#discussion_r1306498052